e2-studio-star-rx72n-firmware/libs/rx_spi_link/src/rx_spi_link.c

```
@@ -0,0 +1,629 @@
+ * @pre link must be initialized
+ * @pre expected_seq matches the last sent frame's sequence
+ * @post No side effects on timeout
+ *
+ * @since Version 1.1.0
+ */
+static rx_err_t internal_wait_for_ack(rx_spi_link_t* link, uint16_t expected_seq)
+{
+    rx_frame_t ack_frame;
+    memset(&ack_frame, 0, sizeof(ack_frame));
+
+    const rx_err_t err = rx_spi_comm_receive(link->spi_handle,
```

Clang (14.0.6)

[warning] variable 'err' is not initialized

(cppcoreguidelines-init-variables)